spark-bigtable_2.12/src/main/scala/com/google/cloud/spark/bigtable/datasources/BigtableSparkConf.scala

```diff
@@ -18,12 +18,15 @@ package com.google.cloud.spark.bigtable.datasources
 
 import org.apache.yetus.audience.InterfaceAudience;
 
-/** This is the bigtable configuration. User can either set them in SparkConf, which
+/** This is the bigtable configuration. Users can either set them in SparkConf, which
   * will take effect globally, or configure it per table, which will overwrite the value
   * set in SparkConf. If not set, the default value will take effect.
   * To access these variables inside Java, use the following import statement:
   * `import static com.google.cloud.spark.bigtable.datasources.BigtableSparkConf.*;`
   * You can then use a syntax like `BIGTABLE_PROJECT_ID()` to access them.
+  * When using RDDs, you can either provide a Map of configs/values or use the
+  * default constructor `new BigtableSparkConf()` and use methods such as
```

Comment:
This doc needs to be updated to use the builder

Reply:
good catch!